lib/src/components/switch/zeta_switch.dart

```diff
@@ -8,16 +8,33 @@ import 'material_switch.dart';
 
 const _sizeAndroid = Size(48, 24);
 const _sizeIOS = Size(56, 32);
+const _sizeWeb = Size(64, 32);
+
+/// Variants of [ZetaSwitch].
+enum ZetaSwitchType {
+  /// 64 x 32
+  web,
+
+  /// 56 x 32
+  android,
+
+  /// 48 x 24
+  ios,
+}
 
```

Comment:
Sizes for `android` and `ios` are the other way around - `android` should be 48 x 24, and `ios` 56 x 32.